communication/src/AsyncTlsConnection.cpp

```diff
@@ -384,103 +385,60 @@ bool AsyncTlsConnection::verifyCertificateServer(asio::ssl::verify_context& ctx)
   if (X509_V_ERR_DEPTH_ZERO_SELF_SIGNED_CERT != X509_STORE_CTX_get_error(ctx.native_handle())) {
     return false;
   }
-  static constexpr size_t SIZE = 512;
-  std::string subject(SIZE, 0);
   X509* cert = X509_STORE_CTX_get_current_cert(ctx.native_handle());
   if (!cert) {
     LOG_WARN(logger_, "No certificate from client");
     return false;
   }
-  X509_NAME_oneline(X509_get_subject_name(cert), subject.data(), SIZE);
-  auto [valid, peer_id] = checkCertificate(cert, "client", std::string(subject), std::nullopt);
+  auto [valid, peer_id] = checkCertificate(cert, std::nullopt);
   peer_id_ = peer_id;
   return valid;
 }
 
 std::pair<bool, NodeNum> AsyncTlsConnection::checkCertificate(X509* receivedCert,
-                                                              std::string connectionType,
-                                                              const std::string& subject,
                                                               std::optional<NodeNum> expectedPeerId) {
```

Comment:
Given that you touch this function, would you mind changing the variables to snake_case as well?

Reply:
fixed